### PR TITLE
Set grace_period to 2h for repo openshift-tests-private & verification-tests

### PR DESCRIPTION
**Pull request by liangxia**

### CODE REVIEW

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-ibmcloud-ipi-tp-f28-destructive

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -354,6 +355,7 @@ plank:
         mediaTypes:
           log: text/plain
       gcs_credentials_secret: gce-sa-credentials-gcs-qe-private-deck
+      grace_period: 2h0m0s
```
Comment (MayXuQQ) [inline core-services/prow/02_config/_config.yaml:358]: can we change the global value?

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-ibmcloud-ipi-mini-perm-f28-destructive-debug

File: core-services/prow/02_config/_config.yaml (hunk 1 above)
Comment (liangxia) [reply core-services/prow/02_config/_config.yaml:358]: Change global value will impact all jobs, while change here will limit the impact to our jobs in openshift-tests-private and verification-tests.

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-ibmcloud-ipi-mini-perm-f28-destructive-debug

**Comment by MayXuQQ** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-ibmcloud-ipi-tp-f28-destructive

**Comment by droslean** [comment]
I am not sure that this will solve the problem. 

/lgtm
/approve

**Pull request merged**